sklearn/ensemble/bagging.py

```diff
@@ -33,6 +33,33 @@
 MAX_INT = np.iinfo(np.int32).max
 
 
+def _draw_indices(random_state, bootstrap, n_pop, n_samp):
```

Comment:
ok so instead of 
`def _draw_indices(random_state, bootstrap, n_pop, n_samp):`
use
`def _draw_indices(random_state, bootstrap, n_population, n_samples):`
?